server/src/providers/blockchain.ts

```diff
@@ -163,10 +169,14 @@ export class blockchain {
 
     if (wallets) {
       for (let index in wallets) {
-        if (this.isClientWallet(wallets[index].address)) {
-          wallets[index].countTx = await transactionsProvider.getCountByWallet(wallets[index].address);
-          await walletsProvider.set(wallets[index]);
-        }
+        wallets[index].balance = await this.getAddressBalance(wallets[index].address);
+        await walletsProvider.set(wallets[index]);
+      }
+    }
+
+    if (aboutTXs) {
+      for (let index in aboutTXs) {
+        await AboutTransactionsProvider.updateOrSet(aboutTXs[index]);
```

Comment:
maybe use 
https://docs.mongodb.com/manual/reference/method/db.collection.bulkWrite/

Reply:
done

Reply:
use **bulkWrite**  for save txInfo and wallet